ui/components/app/snaps/snap-ui-renderer/index.scss

```diff
@@ -1,17 +1,24 @@
 @use "design-system";
 
 .snap-ui-renderer {
+  $height-screen-sm-min: 720px;
   $width-screen-sm-min: 85vw;
   $width-screen-md-min: 80vw;
   $width-screen-lg-min: 62vw;
 
   &__container {
+    height: 100%;
+
     & > *:first-child {
       gap: 16px;
       margin: 16px;
     }
   }
 
+  &__content {
+    flex: 1 1 auto;
```

Comment:
Not clear if this is the cause, but there is still a regression in the scroll behavior where the margin at the bottom isn't scrollable
![image](https://github.com/user-attachments/assets/2dabbc5b-aeb2-4b39-aff4-993fece700f7)

Reply:
thanks for the catch! updated [de06e18](https://github.com/MetaMask/metamask-extension/pull/31454/commits/de06e186fd48adcf09fd8eb354ea9a748670fa70)